streaming/update-rendition.go

```diff
@@ -0,0 +1,14 @@
+package streaming
+
+func GetMimeType(renditionType string) string {
+	switch renditionType {
+	case "video":
+		return "video/mp4"
+	case "audio":
+		return "audio/mp4"
+	default:
+		return "application/mp4"
+
+	}
```

Comment:
I think it's better to create like this
```suggestion

type RenditionType string
consts (
      VideoRenditionType RenditionType = "video"
      AudioRenditionType RenditionType = "audio"
)

type MimeType string
consts (
...
)

var renditionToMimeType = map[RenditionType]MimeType{
...
}

func GetMimeType(renditionType RenditionType) string {
            if value, ok := renditionToMimeType[renditionType]; ok {
                        return string(value)
            }
            return "application/mp4"
```

Reply:
Solved.